instance/config.py

```diff
@@ -0,0 +1,43 @@
+# config file
+
+import os
+
+SECRET_KEY = "Mysecret"
```

Comment:
Hey, allowing important credentials such as  `secret_key` to a public repo is not considered a good practice.